### PR TITLE
Add landcover 2001-2020

### PR DESCRIPTION
**Pull request by vlulla**
Related: https://github.com/NASA-IMPACT/veda-data-pipelines/issues/226 

- Needs legend class names and/or color preferences
- Any specific image for cover?

### CODE REVIEW

**Comment by vlulla** [comment]
I'm not sure why the deploy preview link does not work. The local yarn seems to work for me.

**Comment by vlulla** [comment]
Setting the `"single_datetime": "2020-01-01T00:00:00Z"` instead of `"datetime_range": "year"` in https://github.com/NASA-IMPACT/veda-data-pipelines/blob/cafdec337a20c51b2242422bce84b7c6f1ceb03e/data/step_function_inputs/bangladesh-landcover-2001-2020.json#L17 seems to have fixed the netlify preview link.

**Comment by j08lue** [comment]
Nice things are coming together! 🚀

I can see we still have an issue that the classes are not rendered nicely at overview level, only when you zoom in to full data resolution:
image

But I guess you will publish now anyways and raise a QA issue for later?

**Comment by vlulla** [comment]
@j08lue you are correct! It is unclear to us (me, @moradology, and @ividito), even after multiple discussions and pairing sessions, what is the underlying cause of these rendering issues. It appears that there is something about these tifs that is not playing nicely with the titiler but we are unable to pinpoint what exactly that is!

File: datasets/bangladesh-landcover-2001-2020.data.mdx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+id: bangladesh-landcover-2001-2020
+name: 'Annual land cover maps for 2001 and 2020'
+description: 'Annual land cover maps for 2001 and 2020 (Bangladesh)'
+media:
+  src: ::file ./zz_placeholder.png
```
Comment (slesaad) [inline datasets/bangladesh-landcover-2001-2020.data.mdx:6]: Could use this image - https://unsplash.com/photos/d59NHNtT_Ss. 
We ideally want the cover images to be less than an MB. Here's a [guide](https://github.com/NASA-IMPACT/delta-config/blob/4b7d21655006a2d9056edd234ad46dd04dba69ee/docs/frontmatter/media.md#media) to how that can be achieved.
Don't forget to give proper attribution

```yaml
  author:
    name: Marek Piwnicki
    url: https://unsplash.com/photos/WiZOyYqzUss
```

File: datasets/bangladesh-landcover-2001-2020.data.mdx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+dataset (MCD12Q1 V6, dataset link:
+[https://lpdaac.usgs.gov/products/mcd12q1v006/](https://lpdaac.usgs.gov/products/mcd12q1v006/)).
+The actual data product provides global land cover types at yearly intervals
+(2001-2020) at 500 meters with six different types of land cover
+classification. Among six different schemes, The International
+Geosphere–Biosphere Programme (IGBP) land cover classification selected and
+further simplified to dominant land cover classes (water, urban, cropland,
+native vegetation) for two different years to illustrate the changes in land
+use and land cover of the country.
+</Prose>
```
Comment (slesaad) [inline datasets/bangladesh-landcover-2001-2020.data.mdx:59]: indent?

File: datasets/bangladesh-landcover-2001-2020.data.mdx (hunk 1 above)
Comment (vlulla) [reply datasets/bangladesh-landcover-2001-2020.data.mdx]: Done! Can you please check if it looks ok?

File: datasets/bangladesh-landcover-2001-2020.data.mdx (hunk 2 above)
Comment (vlulla) [reply datasets/bangladesh-landcover-2001-2020.data.mdx]: Done! Can you please check?

File: datasets/bangladesh-landcover-2001-2020.data.mdx [hunk 3]
```diff
@@ -0,0 +1,60 @@
+---
+id: bangladesh-landcover-2001-2020
+name: 'Annual land cover maps for 2001 and 2020'
```
[the remark this answers is not in the document]
Comment (vlulla) [reply datasets/bangladesh-landcover-2001-2020.data.mdx]: Done!

**Review by slesaad — APPROVED** [review]
lgtm

**Comment by vlulla** [comment]
So what do I do next?

**Comment by slesaad** [comment]
@vlulla merge the PR and you should be done! 🎉

**Pull request merged**